src/scf-release/src/acceptance-tests-brain/test-scripts/testutils.rb

```diff
@@ -150,3 +150,27 @@ def wait_for_namespace(namespace, sleep_duration=10)
         sleep sleep_duration
     end
 end
+
+def wait_for_async_service_operation(service_instance_name, retries=0)
+    service_instance_guid = capture("cf service --guid #{service_instance_name}")
+    return { :success => false, :reason => :not_found } if service_instance_guid == 'FAILED'
```

Comment:
FWIW, this is the same as `{success: false, reason: :not_found}`

Reply:
Thanks for pointing that out.